openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KStream.java

```diff
@@ -115,7 +115,7 @@ public void startBroadcast(final StreamDesc sd, final String sdpOffer, Runnable
 		if ((sdpOffer.indexOf("m=audio") > -1 && !hasAudio)
 				|| (sdpOffer.indexOf("m=video") > -1 && !hasVideo && StreamType.SCREEN != streamType))
 		{
-			log.warn("Broadcast started without enough rights");
```

Comment:
do we really need both sid and uid in all these log statements?

Reply:
Yeah, I found during performance testing it is incredible hard trace back the individual client logs to server logs.

Cause once you have 100++ users in parallel on the server, just logged "what" happens does not give you any clue "who" has done it.

And often you end up with 4 different scenarios across 100 users. So the only other chance to correlate server side events to client side events is using timestamps, which is very hard.

Ideally I would like to see a unique identifier for each log statement, so you can search for "uid" or "sid" in the logs and if you do that you can filter all logs of this particular user session.

This will become even more important once you cluster and you have events distributed across multiple servers. It will be incredible hard to debug issues on a running system with 100++ users using it and trying to pinpoint individual logs to individual issues without having unique identifiers logged in every log statement.